instagram-app/src/components/PostContainer/PostsPage.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { CommentSection } from '../CommentSection/CommentSection'
+import styled from 'styled-components'
+
+const Div = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin: 10px;
+    padding: 10px;
+
+`
+
+const Img = styled.img`
+    flex-direction: row;
+    width: 25px;
+    height: 25px;  
+`
+
+export class PostsPage extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            likes: this.props.likes.likes,
+            comments: this.props.data.comments,
+            comment: ''
+        }
+        console.log(props.logout)
+    }
+    addLikes = () => {
+        this.setState((prevState => {
+            return {
+                likes: prevState.likes + 1
+            }
+        }))
+    }
+    handleChange = (event) => {
+        this.setState({ comment: event.target.value })
+    }
+    addComments = (event) => {
+
+        event.preventDefault()
+        const NewComment = {
+            username: 'Matt',
+            text: this.state.comment,
+        }
+        this.setState({ comments: this.state.comments.concat(NewComment) })
```

Comment:
Here we don't require more than one `setState` method, we can put both the `comments` and `comment` properties in the same object. 👍